### ISSUE TITLE
Umbrella: Trello-like List of Calendars

### ISSUE THREAD

**Issue by Restuta** [Restuta/rcn.io#207]
rcn.io/calendars

Should show a list of all calendars we have in trello-like way: 
<img width="1562" alt="trello" src="https://cloud.githubusercontent.com/assets/377261/18370649/891f0f8e-75e3-11e6-9517-1f33e7d80bc6.png">

**Comment by Me4ta** [comment]
Do you mean like a github calendar which you showed to me?

**Comment by Restuta** [comment]
@Me4ta now I think that options is ideal, this one is quicker, so we may start with it.

**Comment by AlexanderKapelyukhovskiy** [comment]
@Restuta Please give more details about task. For now I can see only 3 calendars `ncnca-2016`, `ncnca-2017` and `norcal-mtb`. Do you want to show links to them in some form (trello-like)?

**Comment by Restuta** [comment]
@AlexanderKapelyukhovskiy yes, just like on trello, I can design a quick mockup, but you can start by just creating a view that would point to '/calendars' and show 3 of them (actually 4, one is hidden) under two different categories:

"Official NCNCNA Calendars"

"Non-official Calendars"

**Comment by Restuta** [comment]
In future this page would have two levels "States" (only ncnca now) and "official and non-official"

**Comment by Restuta** [comment]
Alternative exploration of design:

<img width="815" alt="calendar info" src="https://user-images.githubusercontent.com/377261/35460688-be66b16c-0299-11e8-8344-88bc11a5e787.png">
